ScheduleFlow.py

```diff
@@ -593,28 +600,49 @@ def __str__(self):
     def __repr__(self):
         return "Batch "+super(BatchScheduler, self).__repr__()
 
-    def get_batch_jobs(self):
-        ''' Method that returns the first batch_size jobs in the waiting
-        queue ordered by their submission time '''
+    def __get_jobs(self, queue, batch_size):
```

Comment:
The new `__get_job` function changes the order of the jobs selected for the reservation phase compared to the old `get_batch_jobs` function

When simulating the workload from 2018-11-26 on Stempede, 64 jobs are kept in the waiting queue while 334 are running. On each job end the 64 jobs are being investigated for backfill (in average 3 seconds per investigation) causing a 20 minute delay compared to the old code.

The overall schedule improves system utilization and job response time